src/message/tags.rs

```diff
@@ -111,7 +110,7 @@ impl From<HashMap<String, Option<String>>> for IRCTags {
 impl AsRawIRC for IRCTags {
     fn format_as_raw_irc(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         let mut add_separator = false;
-        for (key, value) in self.0.iter().sorted() {
+        for (key, value) in self.0.iter() {
```

Comment:
I've removed the sorted here as the order does not seem to matter according to the [ircv3 specs](https://ircv3.net/specs/extensions/message-tags.html).

Reply:
You're right, I think that should be fine. I cannot remember why I added that `.sorted()` - perhaps to make the output stable so you could unit-test it more easily?